constants.js

```diff
@@ -68,23 +68,22 @@ export default {
         .update('', 'binary').digest('hex'),
 
     // Queries supported by AWS that we do not currently support.
-    unsupportedQueries: [
-        'accelerate',
-        'analytics',
-        'inventory',
-        'lifecycle',
-        'list-type',
-        'logging',
-        'metrics',
-        'notification',
-        'policy',
-        'replication',
-        'requestPayment',
-        'restore',
-        'tagging',
-        'torrent',
-        'versions',
-    ],
+    unsupportedQueries: {
```

Comment:
We have a commit that gathers all the unrelated fixes and optimizations prior to versioning (currently 4c7ee0d , named `clean: some optimization and dev dependencies`) - I moved this change to there.